terraform/README.md

```diff
@@ -16,18 +16,22 @@ rather serve as a building block for higher level modules.
 - **output.tf** - Responsible for integrating the module with other Terraform modules, primarily
   by defining potential integration endpoints (charm integrations), but also by exposing
   the application name.
-- **terraform.tf** - Defines the Terraform provider.
+- **versions.tf** - Defines the Terraform provider version.
 - 
 ## Using sdcore-amf-k8s base module in higher level modules
 
 If you want to use `sdcore-amf-k8s` base module as part of your Terraform module, import it
 like shown below:
 
 ```text
+data "juju_model" "my_model" {
+  name = "my_model_name"
```

Comment:
Shouldn't we use `var.model` here? And then, in line 43, use the reference to the object just like we're doing in line 34?